mailbox/caching/pom.xml

```diff
@@ -31,6 +31,8 @@
     <packaging>bundle</packaging>
     <name>Apache James :: Mailbox :: Caching</name>
 
+    <description>JAMES-2703 This maven module is deprecated and will be removed straight after upcoming James 3.4.0 release, unless it finds a maintainer</description>
+
```

Comment:
no explanation of why and what should be used instead?

Reply:
sorry, but can you add this commit in the java classes you deprecate also?

Reply:
Done